src/Acuminator/Acuminator.Analyzers/StaticAnalysis/Descriptors.cs

```diff
@@ -193,6 +193,26 @@ public static class Descriptors
 			Rule("PX1032", nameof(Resources.PX1032Title).GetLocalized(), Category.Default, DiagnosticSeverity.Error, 
 				DiagnosticsShortName.PX1032);
 
+		public static DiagnosticDescriptor PX1033_MissingDacPrimaryKeyDeclaration { get; } =
+			Rule("PX1033", nameof(Resources.PX1033Title).GetLocalized(), Category.Default, DiagnosticSeverity.Info,
+				DiagnosticsShortName.PX1033);
+
+		public static DiagnosticDescriptor PX1034_MissingDacForeignKeyDeclaration { get; } =
+			Rule("PX1034", nameof(Resources.PX1034Title).GetLocalized(), Category.Default, DiagnosticSeverity.Info,
+				DiagnosticsShortName.PX1034);
+
+		public static DiagnosticDescriptor PX1035_MultiplePrimaryKeyDeclarationsInDac { get; } =
```

Comment:
I think that PX1035 and PX1036 should be warnings. We don't want to force the developers to add PK/FK everywhere but in the case when they added it, PK/FK should be defined in the right way.

Reply:
Ok. Should I write an email to them or ask in comments to the ATR-564 item?

Reply:
Either way is fine; please emphasize the fact that this item is special since the main diagnostic is informational and intended to help all application teams with PK/FK implementation that is planned for Q3.